apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.cs

```diff
@@ -12,9 +12,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-namespace Google.Cloud.Spanner.Data.Tests.V1
+using Google.Cloud.Spanner.V1.Internal.Logging;
+using Google.Protobuf;
+using Moq;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Xunit;
+using static Google.Cloud.Spanner.V1.TransactionOptions;
+
+namespace Google.Cloud.Spanner.V1.PoolRewrite.Tests
 {
     public partial class SessionPoolTests
     {
+        private static readonly SessionName s_sampleSessionName = new SessionName("project", "instance", "database", "session");
+        private static readonly ByteString s_sampleTransactionId = ByteString.CopyFromUtf8("transaction-id");
+
+        [Fact]
+        public void RecreateSession()
+        {
+            var logger = new InMemoryLogger();
+            var mock = new Mock<SpannerClient>();
```

Comment:
Maybe Strict, just to make sure that nothing gets called. I feel Strict should be default in Mock.

Reply:
Cool!